### PR TITLE
InfoList.js

### PR DESCRIPTION
**Pull request by Batuhanbyr**
## InfoList.js has been changed and the review part is complete

### CODE REVIEW

File: webapp/src/components/Filters.js
Comment (FdezAriasSara) [inline webapp/src/components/Filters.js:1]: thank you so much i just noticed that you adapted the icon to appear on the right side, i didnt know how i could do it :) 
image

File: webapp/src/components/InfoList.js
Comment (FdezAriasSara) [inline webapp/src/components/InfoList.js:1]: image
image

I LOVE IT!

image


The only thing i would say is to make the dialogue a little bit bigger so you can see the comment when writting
image


if you want to do it you can include it in the next dont worry :)

Comment (Batuhanbyr) [reply webapp/src/components/InfoList.js:1]: thank you so much, ı will try to do that :)

File: webapp/src/components/Filters.js
Comment (Batuhanbyr) [reply webapp/src/components/Filters.js:1]: thank you :))

**Review by FdezAriasSara — APPROVED** [review]
Everything works and its really nice.

Batu enhanced my filer interface since  i wasn't able to make the closing icon appear where i wanted it.


https://user-images.githubusercontent.com/79209608/232106309-492f152c-83e1-438b-aeb0-815aecbd9200.mp4

**Pull request merged**